src/encoded/tests/test_ingestion_message_handler_decorator.py

```diff
@@ -0,0 +1,236 @@
+import pytest
+import re
+from ..ingestion.ingestion_message import IngestionMessage
+from ..ingestion.ingestion_listener_base import IngestionListenerBase
+from ..ingestion.ingestion_message_handler_decorator import (
+    call_ingestion_message_handler,
+    ingestion_message_handler,
+    clear_ingestion_message_handlers_for_testing,
+)
+
+
+class IngestionListener(IngestionListenerBase):
+    pass  # dummy
+
+
+SOME_UUID = "some-uuid-xyzzy"
+INGESTION_LISTENER = IngestionListener()
+INGESTION_TYPE_VCF = "vcf"
```

Comment:
References to VCF I think can be removed but not harmful to test.

Reply:
Will probably just keep it as I think this will be factored out into common place and shared between cgap-portal and fourfront.